docs/connect.rst

```diff
@@ -96,10 +96,10 @@ Use the standard the `DBAL`_ or `Doctrine ORM`_ documentation for the rest of
 your setup process.
 
 .. _database user: https://crate.io/docs/crate/reference/en/latest/admin/user-management.html
-.. _DBAL documentation: https://www.doctrine-project.org/projects/doctrine-dbal/en/2.7/index.html
-.. _DBAL: https://www.doctrine-project.org/projects/doctrine-dbal/en/2.7/index.html
-.. _Doctrine provided example: https://www.doctrine-project.org/projects/doctrine-orm/en/2.6/reference/configuration.html#obtaining-an-entitymanager
+.. _DBAL documentation: https://www.doctrine-project.org/projects/doctrine-dbal/en/2.10/index.html
```

Comment:
Is it worth referencing the same link twice once under `DBAL documentation` and `DBAL` ? Maybe on reference would be enough ?

Reply:
They are now pointing correctly to 2 different targets.

Reply:
Great, thank you!